tests/unit/database_model_generators/service.py

```diff
@@ -0,0 +1,22 @@
+"""
+
+"""
+from topchef.database.models.service import Service
```

Comment:
Provide an ``__all__`` variable as well in order to mark what types are to be exported from here.